cmake/thirdparty/SetupMPI.cmake

```diff
@@ -136,6 +136,15 @@ if (BLT_MPI_LINK_FLAGS)
     set(_mpi_link_flags ${BLT_MPI_LINK_FLAGS})
 endif()
 
+#
+# We use `LINK_OPTIONS` for CMake 3.13 and beyond.
+# To make sure that de-duping doesn't undermine our MPI flags
+# use a string with SHELL: prefix
+#
+if( ${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.13.0" )
```

Comment:
I know you write perfect code but can you add a note in RELEASE_NOTES.md so that if someone's use case blows up because of this maybe they will see the note about it.

Reply:
Thanks for the reminder, I added a note to rel notes.